src/overlays/actors/ovl_Obj_Spidertent/z_obj_spidertent.c

```diff
@@ -28,118 +35,789 @@ const ActorInit Obj_Spidertent_InitVars = {
+            { -60.900001525878906f, 0.0f, 59.099998474121094f },
+            { 0.8999999761581421f, 30.0f, -2.799999952316284f } } },
+    },
     {
-        { ELEMTYPE_UNK4, { 0x00000000, 0x00, 0x00 }, { 0x00000800, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
-        { { { 17.799999237060547f, 0.0f, 81.5999984741211f }, { 51.599998474121094f, 0.0f, 14.100000381469727f }, { 0.8999999761581421f, 30.0f, -2.799999952316284f } } },
+        {
+            ELEMTYPE_UNK4,
+            { 0x00000000, 0x00, 0x00 },
+            { 0x00000800, 0x00, 0x00 },
+            TOUCH_NONE | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_NONE,
+        },
+        { { { 17.799999237060547f, 0.0f, -87.19999694824219f },
+            { -83.4000015258789f, 0.0f, -19.700000762939453f },
+            { 0.8999999761581421f, 30.0f, -2.799999952316284f } } },
     },
     {
-        { ELEMTYPE_UNK4, { 0x00000000, 0x00, 0x00 }, { 0x00000800, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
-        { { { -60.900001525878906f, 0.0f, 59.099998474121094f }, { 17.799999237060547f, 0.0f, 81.5999984741211f }, { 0.8999999761581421f, 30.0f, -2.799999952316284f } } },
+        {
+            ELEMTYPE_UNK4,
+            { 0x00000000, 0x00, 0x00 },
+            { 0x00000800, 0x00, 0x00 },
+            TOUCH_NONE | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_NONE,
+        },
+        { { { 57.20000076293945f, 0.0f, -47.79999923706055f },
+            { 17.799999237060547f, 0.0f, -87.19999694824219f },
+            { 0.8999999761581421f, 30.0f, -2.799999952316284f } } },
     },
     {
-        { ELEMTYPE_UNK4, { 0x00000000, 0x00, 0x00 }, { 0x00000800, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
-        { { { -83.4000015258789f, 0.0f, -19.700000762939453f }, { -60.900001525878906f, 0.0f, 59.099998474121094f }, { 0.8999999761581421f, 30.0f, -2.799999952316284f } } },
+        {
+            ELEMTYPE_UNK4,
+            { 0x00000000, 0x00, 0x00 },
+            { 0x00000800, 0x00, 0x00 },
+            TOUCH_NONE | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_NONE,
+        },
+        { { { 51.599998474121094f, 0.0f, 14.100000381469727f },
+            { 57.20000076293945f, 0.0f, -47.79999923706055f },
+            { 0.8999999761581421f, 30.0f, -2.799999952316284f } } },
     },
+};
+
+static ColliderTrisInit sTrisInit2 = {
     {
-        { ELEMTYPE_UNK4, { 0x00000000, 0x00, 0x00 }, { 0x00000800, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
-        { { { 17.799999237060547f, 0.0f, -87.19999694824219f }, { -83.4000015258789f, 0.0f, -19.700000762939453f }, { 0.8999999761581421f, 30.0f, -2.799999952316284f } } },
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_NONE,
+        OC2_NONE,
+        COLSHAPE_TRIS,
     },
+    6,
+    sTrisElementsInit2,
+};
+
+typedef struct {
+    /* 0x00 */ UNK_PTR unk_00;
```

Comment:
```suggestion
    /* 0x00 */ Gfx* unk_00;
```
Based on the data below, looks to a DL pointer

Reply:
Agreed. Please update.